Gems/Prefab/PrefabBuilder/PrefabGroup/PrefabGroupBehavior.cpp

```diff
@@ -155,13 +182,112 @@ namespace AZ::SceneAPI::Behaviors
             // indicate the transform should not be set to a default value
             for( const auto& meshIndex : meshIndexContainer)
             {
-                MeshTransformPair pair{ meshIndex, Containers::SceneGraph::NodeIndex{} };
-                meshTransformMap.emplace(MeshTransformEntry{ graph.GetNodeParent(meshIndex), AZStd::move(pair) });
+                MeshNodeData meshNodeData { meshIndex, Containers::SceneGraph::NodeIndex{} };
+                AssignCustomPropertyMapIndex(meshNodeData, graph, meshIndex);
+                meshTransformMap.emplace(MeshTransformEntry{ graph.GetNodeParent(meshIndex), AZStd::move(meshNodeData) });
             }
 
             return meshTransformMap;
         }
 
+        bool AddEditorMaterialComponent(const AZ::EntityId& entityId, const DataTypes::ICustomPropertyData& propertyData)
+        {
+            const auto propertyMaterialPathIterator = propertyData.GetPropertyMap().find("o3de.default.material");
+            if (propertyMaterialPathIterator == propertyData.GetPropertyMap().end())
+            {
+                // skip these assignment since the default material override was not provided
+                return true;
+            }
+
+            const AZStd::any& propertyMaterialPath = propertyMaterialPathIterator->second;
+            if (propertyMaterialPath.empty() || propertyMaterialPath.is<AZStd::string>() == false)
+            {
+                AZ_Error("prefab", false, "Must be a string");
```

Comment:
Can you make this error a bit more descriptive? If this triggers it might not be clear what specifically went wrong and how to fix it.